### PR TITLE
Changed contributors list

### PR DESCRIPTION
**Pull request by Asfalot**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,12 +9,8 @@
...
     },
     {
-      "name": "Alexander Vilko",
-      "email": "alexander.vilko@djangostars.com"
-    },
-    {
-      "name": "Denis Podlesniy",
-      "email": "denis.podlesniy@djangostars.com"
+      "name": "Oleksii Onyshchenko",
+      "email": "asfalot1111@gmail.com"
```
Comment (legendar) [inline package.json]: why you remove other contributors ?

Comment (romanosipenko) [inline package.json]: Also, use your work email!!!

Comment (romanosipenko) [inline package.json]: @legendar you too! :)

**Comment by legendar** [comment]
closed in #65

**Pull request closed without merging**